wordpress-mobile/AztecEditor-iOS
# Media formatters refactor.

This is way way way late, but after banging my head against Aztec for a week and then being sick, I finally managed to get something working. 

This cleans up how Image/Video Formatters work, by splitting them into their own thing — ElementConverters, since, well, they don't really format anything. 

Fixes #841, will need to updated after #847 lands.

To test:
Check if the project compile and unit test are ok.
Start the app, check interactions with image and video are working ok.
